clients/unshaded/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -138,6 +142,23 @@
   }
 
   /**
+   * An alternative to construct a RemoteBlockInstream, bypassing all the checks.
+   * The returned RemoteBlockInStream can be used to call {#link #readRemoteByteBuffer}.
+   * TODO: remove this constructor, and modify all places using this constructor.
+   *
+   * @param file, any, could be null
+   * @param readType, any type
+   * @param blockIndex, any index
+   * @param ufsConf, any ufs configuration
+   * @param tachyonConf, any
+   * @param addFlag, add another field so that this constructor differentiates
+   */
+  public RemoteBlockInStream(TachyonFile file, ReadType readType, int blockIndex, Object ufsConf,
```

Comment:
Does the static method also need to call `readRemoteByteBuffer`? If so, we have to make `readRemoteByteBuffer` non-static, which is opposite of what we are doing; If not, that static method is essentially a wrapper of the "fake" constructor, and we can not prevent people using it. The upside though is that we can give a better name for the static method, e.g. `getEmptyStream`, or `getFakeStream`. Do you think it is better?